pandas/core/series.py

```diff
@@ -1372,6 +1379,8 @@ def reset_index(self, level=None, drop=False, name=lib.no_default, inplace=False
             when `drop` is True.
         inplace : bool, default False
             Modify the Series in place (do not create a new object).
+        allow_duplicates : bool, default False
```

Comment:
None default right? also add that this looks at the option

add versionadded 1.4

Reply:
It is DataFrame. not Series,  where it looks at its flag (not option)